packages/cli/src/generator.ts

```diff
@@ -176,6 +126,8 @@ export async function queryToTypeDeclarations(
     }
   }
 
+  await types.check();
```

Comment:
This throws if there was an error (which currently leads to #53). Just thinking about the pattern for error handling you describe a preference for there. Perhaps this should just be logging the errors. Unmapped types already output `never`.

Reply:
Lets remove this call for now and revisit error handling in #53